pkg/parser/cse_cmd.sh.gtpl

```diff
@@ -1,3 +1,17 @@
-#!/bin/bash
-echo "hello"
-#This is a simple placeholder script, but will eventually resemble what cse_cmd.sh looks today
\ No newline at end of file
+ADMINUSER="{{.LinuxAdminUsername}}"
+PROVISION_OUTPUT="{{.ProvisionOutput}}"
+{{if .CustomCloudConfig.IsCustomCloud}}
+REPO_DEPOT_ENDPOINT="{{.RepoDepotEndpoint}}"
+{{/* need to properly add {{GetInitAKSCustomCloudFilepath}} >> /var/log/azure/cluster-provision.log 2>&1; */}}
+{{end}}
+MOBY_VERSION="{{.MobyVersion}}"
+TENANT_ID="{{.TenantId}}"
+KUBERNETES_VERSION="{{.KubernetesVersion}}"
+HYPERKUBE_URL="{{.HyperkubeUrl}}"
+KUBE_BINARY_URL="{{.KubeBinaryUrl}}"
+CUSTOM_KUBE_BINARY_URL="{{.CustomKubeBinaryUrl}}"
+KUBEPROXY_URL="{{.KubeproxyUrl}}"
+
```

Comment:
extra line for variables between? we can remove it for now and order it with cse_cmd.sh later

Reply:
updated